### ISSUE TITLE
dokuwiki references. 

### ISSUE THREAD

**Issue by leamas** [opencpn-manuals/plugins#17]
There is  a few references to the dokuwiki plugin manual. These should be fixed by moving the referenced content to Antora.

Here are also references to the Antora User Manual. These are not a problem, the UM llives in the dokuwiki for now.

Currently, we have 9 references: 

    $ grep -r dokuwiki sources | wc
    9      35    1781

Of these, quite a few comes from watchdog: 

```
$ grep -r dokuwiki sources | grep watchdog
sources/watchdog_pi/manual/watchdog/modules/ROOT/pages/watchdog.adoc:Watchdog communicates with another very useful plug-in, called  https://opencpn.org/wiki/dokuwiki/doku.php?id=opencpn:opencpn_user_manual:plugins:safety:odraw1.6_pi[Ocpn_Draw].
sources/watchdog_pi/manual/watchdog/modules/ROOT/pages/watchdog.adoc:https://opencpn.org/wiki/dokuwiki/doku.php?id=opencpn:opencpn_user_manual:plugins:install_and_enable[Plugin Download, Install and Enable].
sources/watchdog_pi/manual/watchdog/modules/ROOT/pages/watchdog.adoc:See also  https://opencpn.org/wiki/dokuwiki/doku.php?id=opencpn:opencpn_user_manual:plugins:plugin_messaging[Plugin Messaging between ODraw, Watchdog and Weather_routing.]
sources/watchdog_pi/manual/watchdog/modules/ROOT/pages/watchdog.adoc:using the ChartDownloader, see  https://opencpn.org/wiki/dokuwiki/doku.php?id=opencpn:opencpn_user_manual:charts:chart_sources#high_resolution_background_map[High
sources/watchdog_pi/manual/watchdog/modules/ROOT/pages/watchdog.adoc:https://opencpn.org/wiki/dokuwiki/doku.php?id=opencpn:opencpn_user_manual:plugins:other:pypilot

```

First action here  should be a PR against watchdog

**Comment by leamas** [comment]
Filed https://github.com/rgleason/watchdog_pi/pull/29, 5 more references left...

**Comment by leamas** [comment]
Many of  the remaining dokuwiki  references are about "Supplemental hardware" and "Supplemental Software"

@rgleason : Could you possibly make the sources available for these chapters?

**Comment by rgleason** [comment]
Sure, 
How many pages?
Are they essential references?  If they aren't perhaps drop them?
We are taking different pages from Supplimental Hw and Softw and putting them in Miscl.  which seems a little too bad.
Why not just leave the links?  Is that so bad if they stay there?

I suppose I could generate a list....
I am busy on weather_routing right now.  Will try.

PS  I guess I should use 

` grep -r dokuwiki sources | wc`

**Comment by leamas** [comment]
Last pieces fixed in 0447eed. We should add a test for dokuwiki references in the build before closing

**Comment by rgleason** [comment]
Gosh you are fast.

**Comment by leamas** [comment]
Check added and finally fixed in 6877ecf5ebc. All references are gone. Closing
